modules/modal/react/stories/stories_testing.tsx

```diff
@@ -0,0 +1,166 @@
+/// <reference path="../../../../typings.d.ts" />
+import * as React from 'react';
+
+import {Modal, useModal} from '@workday/canvas-kit-react-modal';
+import {Button, DeleteButton} from '@workday/canvas-kit-react-button';
+import {
+  Popup,
+  Popper,
+  usePopup,
+  useCloseOnOutsideClick,
+  useCloseOnEscape,
+} from '@workday/canvas-kit-react-popup';
+
+export default {
+  title: 'Testing|Popups/Modal/React',
```

Comment:
Should this be `Testing|React/Popups/Modal` instead?

Reply:
Ah. Good catch. I didn't see the change to navigation hierarchy here

Reply:
There is still going to be a `Popups` top-level since `Popup Stack` is not React specific... That seems odd... Maybe just `Popup Stack`?

Reply:
Or we could have `Testing|JavaScript/...`